src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -2,21 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Globalization;
 using System.Resources;
 using System.Runtime.CompilerServices;
 using System.Threading;
 
-namespace System
+namespace Microsoft.Data
```

Comment:
Not required change as of this PR, lets keep the changeset specific to Component model annotations. Localization file changes will be done in an isolated activity.

Reply:
done.